Thursday September 26th 2013 4:16 am -04:00,I have some question:(1)Method Description:NoneMessage:ERROR: DisabledBuiltinErrorCalling the open built-in is not permitted.A part of code: def read_words(words_file):        words_list =     file = open(words_filer)    #......        return words_list(2)WHY?Traceback (most recent call last):  File C:\Python33\a3_driver.py line 88 in <module>    words = a3.read_words(words_file)  File C:\Python33\a3.py line 183 in read_words    file = open(words_filer)TypeError: invalid file: <_io.TextIOWrapper name=C:/Python33/wordlist1.txt mode=r encoding=cp950>, Unresolved,0
